samples/boards/stm32/i2c_config/src/main.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/kernel.h>
+
+
+#if DT_NODE_HAS_STATUS(DT_ALIAS(i2c_0), okay)
+#define I2C_DEV_NODE	DT_ALIAS(i2c_0)
+#elif DT_NODE_HAS_STATUS(DT_ALIAS(i2c_1), okay)
+#define I2C_DEV_NODE	DT_ALIAS(i2c_1)
+#elif DT_NODE_HAS_STATUS(DT_ALIAS(i2c_2), okay)
+#define I2C_DEV_NODE	DT_ALIAS(i2c_2)
+#else
+#error "Please set the correct I2C device"
+#endif
```

Comment:
This whiole if/else shouldn't be required, you can impose to use i2c_0 alias

Reply:
sure